src/api/OrganizationApiUtils.ts

```diff
@@ -0,0 +1,47 @@
+export interface Organization {
```

Comment:
przerzucmy to do pliku z typami, mozemy tez uzyc jakiegos narzedzia do mapowania typow z backendu na frontend, znam ReinforcedTypings